docs(angular.md): update to enable NgModule to contain Non-Angular elements (#3764)
microsoft/fast: packages/web-components/fast-foundation/docs/integrations/angular.md

}
```

> To allow NgModule to contain Non-Angular elements named, add  the code in your `app/app.module.ts` file contents with this:

Reviewer: I think you could just say "Allows NgModule..." and there is a double space in ", add  the code".

> To allow NgModule to contain Non-Angular elements named, add  the code in your `app/app.module.ts` file contents with this:

Reviewer 2: ```suggestion
To allow an NgModule to contain Non-Angular element names, add the following code in your `app/app.module.ts` file:
```

```ts 

> import {CUSTOM_ELEMENTS_SCHEMA  } from '@angular/core';

Reviewer: the spaces are off inside the `{..}` brackets.  Prettier may catch, this though I'm not 100% positive

> import {CUSTOM_ELEMENTS_SCHEMA  } from '@angular/core';

Reviewer 2: ```suggestion
import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
```
